graph-manager-docs/source/managed-federation/setup.mdx

```diff
@@ -0,0 +1,134 @@
+
+With managed federation, Graph Manager (_not_ your gateway) is responsible for combining your individual schemas into a single federated schema. Consequently, **each service in your data graph must re-register its schema every time the schema changes.** Otherwise, your gateway can't obtain the latest federated schema from Graph Manager.
+
+The best way to accomplish this is to add the `apollo service:push` call to your continuous delivery pipeline. Every time you push a schema change to production, this call should automatically accompany that push.
+
+For more information on best practices for updating your services, see [Pushing configuration updates safely](./advanced-topics/#pushing-configuration-updates-safely).
+
+## 3. Modify the gateway
+
+Because your gateway isn't responsible for creating your federated schema anymore, you need to modify its initialization.
+
+> This section assumes you are using Apollo Server with the `@apollo/gateway` library as your gateway.
+
+### Remove the `serviceList` argument
+
+A bare-bones initialization of an `ApolloGateway` object looks like this:
+
+```js
+const gateway = new ApolloGateway({
+  serviceList: [
+    { name: 'accounts', url: 'http://localhost:4001' },
+    // Additional services defined here
+  ],
+});
+```
+
+The `serviceList` argument specifies the name and URL for each of your implementing services. With managed federation, your gateway obtains this information from Graph Manager instead.
+
+**Remove the `serviceList` argument from your `ApolloGateway` constructor entirely.**
+
+### Connect the gateway to Graph Manager
+
+Like your implementing services, your gateway uses an API key to identify itself to Graph Manager. Create an API key from your graph's Settings page in the [Graph Manager UI](https://engine.apollographql.com/).
+
+Provide your API key to the gateway in any of the following ways:
+
+* In your gateway's environment, set the `ENGINE_API_KEY` envrionment variable to the value of your API key.
+* In the constructor of `ApolloServer`, provide an `engine` argument with `apiKey` and `schemaTag` fields, like so:
+
+    ```js{9-12}
+    const { ApolloServer } = require('apollo-server');
+    const { ApolloGateway } = require('@apollo/gateway');
+
+    const gateway = new ApolloGateway();
+
+    // Pass the gateway to Apollo Server
+    const server = new ApolloServer({
+      gateway,
+      engine: {
+        apiKey: YOUR_API_KEY,
+        schemaTag: 'staging'
+      },
+      subscriptions: false 
+    });
+
+    server.listen().then(({ url }) => {
+      console.log(`🚀 Server ready at ${url}`);
+    });
+    ```
+
+    The `engine` object is of type `EngineReportingOptions`. [See the API reference for all available fields.](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#enginereportingoptions)
+
+As with your services, you can either remove or replace the `schemaTag` argument when you're ready to deploy to your production environment.
```

Comment:
I would flip this sentence: "When you're ready to deploy to your production environment, ..."